app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = [Person(person["name"], person["age"]) for person in people]
+
+    for person in people:
+        if "wife" in person and person["wife"] is not None:
```

Comment:
consider to use get() instead of checking if key in dictionary

Reply:
Can't remember if we used getters/setters in the course yet. So did w/o one. But yeah, it's better with a getter use. Thank you

Reply:
i meant dictionary.get(), so u don't have to check if key in dictionary